include/z64.h

```diff
@@ -318,7 +318,7 @@ typedef struct {
 typedef struct {
     /* 0x0000 */ View   view;
     /* 0x0128 */ Font   font;
-    /* 0xE2B0 */ void*  textboxSegment; // "fukidashiSegment"
+    /* 0xE2B0 */ u8*    textboxSegment; // "fukidashiSegment"
```

Comment:
does changing all of these to u8* remove the u32 casts in the dma calls?

Reply:
It removes some casts, but `u8*` was also necessary to match some stuff in kaleido scope, so I changed all "segment" pointers to `u8*` for consistency (unless I missed some)